reference_designs/ipu-xrt/eltwise_add/aie2.py

```diff
@@ -0,0 +1,152 @@
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2023 AMD Inc.
+
+import sys
+
+from aie.dialects.aie import *
+from aie.dialects.aiex import *
+from aie.dialects.scf import *
+from aie.extras.context import mlir_mod_ctx
```

Comment:
I think it would be clearer to have something more understandable for beginners with less aggressive `import`.
For example just
```python
from aie.dialects import *
```
that would replace
```python
@core(cores[i], "add.o")
```
by
```python
@aie.core(cores[i], "add.o")
```
and so on.